src/static/css/2019.css

```diff
@@ -199,47 +199,64 @@ blockquote::before {
 }
 
 /* Footer */
-footer hr:last-of-type {
-  display: none;
+footer {
+  display: grid;
```

Comment:
Is grid the best fit for this layout? I would have thought flexbox given the fluidity. Does it work better in your opinion?

Reply:
Thank @rviscomi  for the review!

When I implemented it responsively,  I decided that the next elements would be complicated if flexbox, and that the grid layout would may be simple.

1. copyright and social element order
2. Language switch and navigation element order
3. http archive home logo position

The layout is confirmed with firefox/chrome/safari on mac (and safari on iPhone).

This judgment may be my lack of knowledge.
I am interested in the possibility that flexbox is more suitable.

Reply:
@rviscomi 
I'm sorry, I forgot to issues #235.
I will fix it soon.